contracts/contracts/vault/VaultCore.sol

```diff
@@ -671,6 +659,24 @@ contract VaultCore is VaultStorage {
         return assets[_asset].isSupported;
     }
 
+    function _toUnits(uint256 raw, address _asset)
+        internal
+        view
+        returns (uint256)
+    {
+        if (assets[_asset].hasExchangeRate) {
+            return (raw * IExchangeRateToken(_asset).exchangeRate()) / 1e18;
```

Comment:
For [rEth](https://etherscan.io/address/0xae78736cd615f374d3085123a210448e74fc6393#readContract) it is `getExchangeRate` 

<img width="493" alt="Screenshot 2023-04-03 at 14 39 45" src="https://user-images.githubusercontent.com/579910/229512047-57b06e2a-3cc9-450e-a6c4-b787bb875cee.png">